### PR TITLE
WIP: export ironic and mariadb env vars coming from vars.sh

### PR DESCRIPTION
**Pull request by tuminoid**
TL;DR: Those variables get lost between integration test start and 02 script trying to use them.

I'm testing https://github.com/metal3-io/ironic-image/pull/466 Ironic source build after https://github.com/metal3-io/metal3-dev-env/pull/1330 got merged, and from the log I can see `IRONIC_FROM_SOURCE` is set to `true` in project-infra, it gets into `/tmp/vars.sh` which is sourced in `integration_test.sh` but when the test says `make ci_run` and runs `02` script, which sources `common.sh`, the `IRONIC_FROM_SOURCE` variable is empty, and gets defaulted to `false`. 

https://jenkins.nordix.org/job/metal3_ironic_image_main_integration_test_centos_ironic_from_source/10/consoleFull

### CODE REVIEW

**Comment by tuminoid** [comment]
> /test-ubuntu-integration-main-ironic-source

Does not work from project-infra, so cannot tell if it works or not. Need to fix JJB I guess.
/hold

**Comment by tuminoid** [comment]
Testing random thing.

/test centos-integration-main

**Comment by tuminoid** [comment]
/hold
This needs JJB changes to enable `-ironic-source` from project-infra too, not only from ironic-image.

**Comment by tuminoid** [comment]
Testing unrelated things:

> /test-ubuntu-integration-main 

New regex will trigger it...?

**Comment by tuminoid** [comment]
This has rotten, will abandon for now.

/close
